data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -1240,94 +1506,102 @@
                         name=this_item.id
                         equals=disguise
                     [/variable]
-
                     [then]
                         {CLEAR_VARIABLE this_item}
                     [/then]
                 [/if]
             [/do]
         [/foreach]
-
         [unstore_unit]
             variable=unit
             find_vacant=no
         [/unstore_unit]
+
+        [modify_side]
+            side,controller=2,ai
+        [/modify_side]
     [/event]
 
+# orcish treasury gold, so even if you didn't save anything, you may still be able to start S12 with a little bit
+#define TREASURY_GOLD X Y IMAGE AMOUNT
+    {PLACE_IMAGE {IMAGE} {X} {Y}}
+    {SET_LABEL {X} {Y} _"{AMOUNT} gold"}
     [event]
         name=moveto
 
         [filter]
-            side=1
-            x,y=33,11
+            side,x,y=1,{X},{Y}
         [/filter]
 
-        [filter_condition]
-            [variable]
-                name=stored_player_gold
-                greater_than_equal_to=1
-            [/variable]
-        [/filter_condition]
-
+        {REMOVE_IMAGE $unit.x $unit.y}
+        {REMOVE_LABEL $unit.x $unit.y}
         [message]
-            speaker=unit
-            message= _ "All the gold the orcs took is in this chest!"
-            sound=open-chest.wav
+            speaker,image,sound=narrator,wesnoth-icon.png,gold.ogg
+            message= _ "You’ve pillaged {AMOUNT} orcish gold pieces!"
         [/message]
-
         [gold]
-            side=1
-            amount=$stored_player_gold
+            side,amount=1,{AMOUNT}
         [/gold]
+    [/event]
+#enddef
+    {TREASURY_GOLD 34 10 "items/gold-coins-small.png"  70}
+    {TREASURY_GOLD 35 11 "items/gold-coins-medium.png" 115}
 
-        [message]
-            speaker=narrator
-            message= _ "You’ve regained $stored_player_gold gold!"
-            image=wesnoth-icon.png
-            sound=gold.ogg
-        [/message]
-
-        [remove_item]
-            x,y=33,11
-            image=items/chest-plain-closed.png
-        [/remove_item]
-
-        {PLACE_IMAGE items/chest-plain-open.png 33 11}
+    #---------------------------
+    # RING OF INVISIBILITY
+    #---------------------------
+    [event]
+        name=sighted
 
-        {CLEAR_VARIABLE stored_player_gold}
+        [filter]
+            id=Cowardly Goblin
+        [/filter]
 
-        [show_objectives][/show_objectives]
+        {SCROLL_TO 31 21}
+        [message]
+            speaker=unit
+            message= _ "Oooh, so shiny, so precious, <i>my precious</i>... What a great gift the Chief has given me."
+        [/message]
+        [message]
+            speaker=unit
+            message= _ "Aaah, humans! Not again not again!"
```

Comment:
Comma between the 'not again's.

Reply:
Deliberate in this case - I wanted to make it sound as if the goblin was speaking very quickly and panicky.  Not sure if that gets across or not.

Reply:
Maybe if it was something that happened a lot (eg there's still a comma in the first sentence), but since this is a one-off, it reads like a mistake to me rather than something intentional.

Reply:
I can't think of a better way to convey rapid speech, but the comma _does_ manage to do that. It also manages to look like an error, though, so that's not great.

Does removing the first comma work? I'm not sure if it even works for me, but it sounds like it might work for Wedge at least…

Reply:
I'm not really comfortable with the lack of punctuation at all, but again, if it was something that was consistently conveyed, I could get the gist it was intentional. Since we only have this example, it reads poorly to me.

Reply:
Maybe remove the spaces between the words? i.e. "Notagainnotagain!"